resources/views/layout/metadata.blade.php

```diff
@@ -28,6 +28,18 @@
 <meta name="keywords" content="osu, peppy, ouendan, elite, beat, agents, ds, windows, game, taiko, tatsujin, simulator, sim, xna, ddr, beatmania, osu!, osume">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+@if (isset($opengraph))
+    <meta property="og:site_name" content="{{ 'osu! » '.$opengraph['site_name'] ?? 'osu!' }}">
```

Comment:
this doesn't work (`'x'.$a ?? 'y'`, the `??` is for the entire `'x'.$a`, not just `$a`). Use a separate php block instead. And `site_name` for the key sounds a bit weird (and not quite exactly used as is either). `section` is probably fine.